Sources/TuistLoaderTesting/Utils/Mocks/MockManifestFilesLocator.swift

```diff
@@ -3,18 +3,32 @@ import TSCBasic
 @testable import TuistLoader
 
 public final class MockManifestFilesLocator: ManifestFilesLocating {
-    public var locateStub: [(Manifest, AbsolutePath)]?
-    public var locateArgs: [AbsolutePath] = []
-    public var locateAllStubs: [(Manifest, AbsolutePath)]?
-    public var locateAllArgs: [AbsolutePath] = []
+    public var locateProjectManifestsStub: [(Manifest, AbsolutePath)]?
```

Comment:
I think it's better than having a bunch of these properties to have a more streamlined approach of 
`locateProjectManifestsStub: ((AbsolutePath) -> [(Manifest, AbsolutePath)])?`. It's used like this throughout the project and it's something contributors are already familiar with

Reply:
Actually I have seen a lot of different approaches of writing mocks in Tuist. 

For example: `MockXCFrameworkBuilder`, `MockHelpersDirectoryLocator`, `MockManifestLoader` are written as mine.
For example: `MockGraphContentHasher` have bunch of properties.

And finally: `MockBuildGraphInspector`, `MockCloudHTTPRequestAuthenticator` are written as you suggested.

How should I behave then?

Reply:
Hmm, yeah, you're right, maybe that's just my convention then 😁 

@pepibumur, is there convention we want to stick to when writing mocks? And is that something we want to enforce?

Reply:
Im using MockGenerator. https://github.com/seanhenry/SwiftMockGeneratorForXcode

@pepibumur recommended it to me.

<img width="484" alt="Zrzut ekranu 2020-09-25 o 10 29 18" src="https://user-images.githubusercontent.com/4774319/94245029-fc470700-ff19-11ea-958e-39476503062b.png">